doc/getting_started/getting_started.rst

```diff
@@ -45,7 +45,7 @@ Next, install additional Python packages required by Zephyr in a shell or
 .. code-block:: console
 
    # Linux
-   pip3 install --user -r zephyr/scripts/requirements.txt
+   pip3 install -r --user zephyr/scripts/requirements.txt
```

Comment:
Thanks for the patch! Can you explain the fix in the commit log?

Looking at it, I don't think it is correct. The argument to the -r option is zephyr/scripts/requirements.txt, not --user. What am I missing?

Reply:
The ordering was originally wrong. 
"-r" should be placed right before the path to the requirements.txt to recognize it as such one.

Reply:
I'm confused, didn't we fix this with:

https://github.com/zephyrproject-rtos/zephyr/pull/10823

Reply:
Ok, please ignore this patch, I think I runned into an old documentation and changed it the wrong way.
But please consider about the second patch which named missing packages for Arch Linux.

Reply:
@Allegra42 	do you mind update the commit to drop the pip change, and just have the update for the Arch

Reply:
fixed (and merge) in another PR #10823 already.